src/main/java/io/github/cottonmc/cotton/gui/widget/WDynamicLabel.java

```diff
@@ -18,28 +18,49 @@
  */
 public class WDynamicLabel extends WWidget {
 	protected Supplier<String> text;
-	protected HorizontalAlignment alignment = HorizontalAlignment.LEFT;
```

Comment:
This is a breaking change. The field can be renamed in a major version bump (= Minecraft update), but right now, it can just stay as `alignment`.

Reply:
That's why I kept the methods the same. But I guess the field can be accessed in extended classes, will revert